assets/javascripts/discourse/connectors/user-profile-avatar-img-wrapper/user-profile-avatar-img-wrapper.hbs

```diff
@@ -0,0 +1 @@
+{{animated-bound-avatar this.user "huge"}}
```

Comment:
maybe this is a good opportunity to convert this to gjs and to move/inline that helper implementation here?